doc/usage/restructuredtext/basics.rst

```diff
@@ -258,8 +258,8 @@ target formats (HTML, LaTeX) have a limited supported nesting depth.
 Field Lists
 -----------
 
-Field lists (:duref:`ref <field-lists>`) are sequences of fields marked up like
-this::
+Field lists (:duref:`docutils ref <field-lists>`) are sequences of fields
```

Comment:
We usually use "(ref)" link in this page. It would be better to change them all in once. Anyway, it should be in another PR.

Reply:
If you tell me what you'd like to see I am happy to do it here, to be honest I don't understand the difference between `duref` and `ref` so I was just trying to copy what else was there